modules/dnn/src/layers/fast_convolution/fast_convolution.simd.hpp

```diff
@@ -0,0 +1,346 @@
+
+            a0 = v_setall_f32(a[2]);
+            c12 = v_fma(b0, a0, c12);
+            c13 = v_fma(b1, a0, c13);
+            c14 = v_fma(b2, a0, c14);
+            c15 = v_fma(b3, a0, c15);
+            c16 = v_fma(b4, a0, c16);
+            c17 = v_fma(b5, a0, c17);
+
+            a0 = v_setall_f32(a[3]);
+            c18 = v_fma(b0, a0, c18);
+            c19 = v_fma(b1, a0, c19);
+            c20 = v_fma(b2, a0, c20);
+            c21 = v_fma(b3, a0, c21);
+            c22 = v_fma(b4, a0, c22);
+            c23 = v_fma(b5, a0, c23);
+        }
+
+        if (ifActiv) {
+            v_float32x4 vmin = v_setall_f32(minval), vmax = v_setall_f32(maxval);
+            c0 = v_min(v_max(c0, vmin), vmax);
+            c1 = v_min(v_max(c1, vmin), vmax);
+            c2 = v_min(v_max(c2, vmin), vmax);
+            c3 = v_min(v_max(c3, vmin), vmax);
+            c4 = v_min(v_max(c4, vmin), vmax);
+            c5 = v_min(v_max(c5, vmin), vmax);
+            c6 = v_min(v_max(c6, vmin), vmax);
+            c7 = v_min(v_max(c7, vmin), vmax);
+            c8 = v_min(v_max(c8, vmin), vmax);
+            c9 = v_min(v_max(c9, vmin), vmax);
+            c10 = v_min(v_max(c10, vmin), vmax);
+            c11 = v_min(v_max(c11, vmin), vmax);
+            c12 = v_min(v_max(c12, vmin), vmax);
+            c13 = v_min(v_max(c13, vmin), vmax);
+            c14 = v_min(v_max(c14, vmin), vmax);
+            c15 = v_min(v_max(c15, vmin), vmax);
+            c16 = v_min(v_max(c16, vmin), vmax);
+            c17 = v_min(v_max(c17, vmin), vmax);
+            c18 = v_min(v_max(c18, vmin), vmax);
+            c19 = v_min(v_max(c19, vmin), vmax);
+            c20 = v_min(v_max(c20, vmin), vmax);
+            c21 = v_min(v_max(c21, vmin), vmax);
+            c22 = v_min(v_max(c22, vmin), vmax);
+            c23 = v_min(v_max(c23, vmin), vmax);
+        }
+        v_store(c, c0);
+        v_store(c + 4, c1);
+        v_store(c + 8, c2);
+        v_store(c + 12, c3);
+        v_store(c + 16, c4);
+        v_store(c + 20, c5);
+
+        v_store(c + ldc, c6);
+        v_store(c + ldc + 4, c7);
+        v_store(c + ldc + 8, c8);
+        v_store(c + ldc + 12, c9);
+        v_store(c + ldc + 16, c10);
+        v_store(c + ldc + 20, c11);
+
+        v_store(c + ldc * 2, c12);
+        v_store(c + ldc * 2 + 4, c13);
+        v_store(c + ldc * 2 + 8, c14);
+        v_store(c + ldc * 2 + 12, c15);
+        v_store(c + ldc * 2 + 16, c16);
+        v_store(c + ldc * 2 + 20, c17);
+
+        v_store(c + ldc * 3, c18);
+        v_store(c + ldc * 3 + 4, c19);
+        v_store(c + ldc * 3 + 8, c20);
+        v_store(c + ldc * 3 + 12, c21);
+        v_store(c + ldc * 3 + 16, c22);
+        v_store(c + ldc * 3 + 20, c23);
+        return;
+    }
+#endif
+#endif
+    CV_LOG_WARNING(NULL, "Runing at unoptimized code. The combination of FAST_CONV_MR and FAST_CONV_NR is not supported in SIMD128 branch.");
+    conv_blockDefualt(k, a, b, c, ldc,bias, minval, maxval, ifActiv);
```

Comment:
please, remove "return" from the previous section and put `#else #error "..." #endif`, otherwise it looks weird

Reply:
fixed.